gen-linux.sh

```diff
@@ -2,4 +2,4 @@
 set -ex
 
 # Generate project files for Ninja
-cmake --preset linux-ninja-clang
+cmake --preset linux-ninja-clang -B build-linux
```

Comment:
Actually build-linux is not used anymore, just forgot to remove the os specific build folders out of the gitignore, now it's all in a single build folder, revert this one, the rest i think @Zangetsu38 should handle it

Reply:
ah look like windows still has its own folder, just linux uses `build`, while windows uses `build-windows`

Reply:
On it.